OpenQA/Isotovideo/Utils.pm

```diff
@@ -176,7 +176,7 @@ present.
 
 =cut
 
-sub load_test_schedule () {
+sub load_test_schedule (@) {
```

Comment:
Why this one? I think this is only called without any arguments.

Reply:
Well, some tests failed is what people told me :)